config/config_internal_test.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"os"
+	"testing"
+)
+
+func TestSetGetConfiguration(t *testing.T) {
```

Comment:
I think `TestGetConfiguration` is better than `TestSetGetConfiguration` as the function name.

Why not create a file from your code and test it? If you import an external file and test some function, it seems to be dependent on the outside. If you think it's okay to generate the file from code, create an issue and assign it to us. I'll work on it.

Reply:
fixed!